docs/sources/flow/reference/components/prometheus.exporter.azure.md

```diff
@@ -0,0 +1,148 @@
+| `metric_aggregations`         | `list(string)` | Aggregations to apply for the metrics produced.                      |                                                                               | no       |
+| `timespan`                    | `string`       | [ISO8601 Duration][] over which the metrics are being queried.       | `"PT1M"` (1 minute)                                                           | no       |
+| `included_dimensions`         | `list(string)` | List of dimensions to include on the final metrics.                  |                                                                               | no       |
+| `included_resource_tags`      | `list(string)` | List of resource tags to include on the final metrics.               | `["owner"]`                                                                   | no       |
+| `metric_namespace`            | `string`       | Namespace for `resource_type` which have multiple levels of metrics. |                                                                               | no       |
+| `azure_cloud_environment`     | `string`       | Name of the cloud environment to connect to.                         | `"azurecloud"`                                                                | no       |
+| `metric_name_template`        | `string`       | Metric template used to expose the metrics.                          | `"azure_{type}_{metric}_{aggregation}_{unit}"`                                | no       |
+| `metric_help_template`        | `string`       | Description of the metric.                                           | `"Azure metric {metric} for {type} with aggregation {aggregation} as {unit}"` | no       |
+
+The list of available `resource_type` values and their corresponding `metrics` can be found in [Azure Monitor essentials][].
+
+The `resource_graph_query_filter` can be embedded into a template query of the form `Resources | where type =~ "<resource_type>" <resource_graph_query_filter> | project id, tags`.
+
+Valid values for `metric_aggregations` are `minimum`, `maximum`, `average`, `total`, and `count`. If no aggregation is specified, the value is retrieved from the metric. For example the aggregation value of the metric `Availability` in [Microsoft.ClassicStorage/storageAccounts](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-classicstorage-storageaccounts-metrics) is `average`.
+
+Every metric has its own set of dimensions. For example the dimensions for the metric `Availability` in [Microsoft.ClassicStorage/storageAccounts](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-classicstorage-storageaccounts-metrics) are `GeoType`, `ApiName`, and `Authentication`. If a single dimension is requested it will have the name `dimension`. If multiple dimensions are requested they will have the name `dimension<dimension_name>`.
```

Comment:
```suggestion
Every metric has its own set of dimensions. For example, the dimensions for the metric `Availability` in [Microsoft.ClassicStorage/storageAccounts](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-classicstorage-storageaccounts-metrics) are `GeoType`, `ApiName`, and `Authentication`. If a single dimension is requested, it will have the name `dimension`. If multiple dimensions are requested, they will have the name `dimension<dimension_name>`.
```